### PR TITLE
Make template changes for TCG-Storage workgroup

### PR DESCRIPTION
**Pull request by joyshukla**
- Add support for Courier New font
- Add support for storage tables

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -138,6 +138,7 @@ COPY --from=build-texlive /texlivebins /usr/local/texlive
 # Copy only the fonts we're using from the template.
 COPY --from=build-fonts \
     /usr/share/fonts/truetype/msttcorefonts/Arial* \
+    /usr/share/fonts/truetype/msttcorefonts/Courier* \
```
Comment (chrisfenner) [inline Dockerfile:141]: How important is it to specifically use Courier? We use Noto Sans Mono for code (`monospaced`) text currently. However, TCG's templates haven't historically been that picky about monospaced fonts.

File: template/tcg.tex [hunk 2]
```diff
@@ -368,6 +368,9 @@
 \setsansfont{Arial}
 \setmonofont{Noto Sans Mono}
 
+% TCG storage specific customizations
```
Comment (chrisfenner) [inline template/tcg.tex:371]: Can you create a GH issue in this repository and reference it with a TODO-style comment that we would like to generalize and migrate features out of this file and into the general template over time?

File: template/tcg-storage.tex [hunk 3]
```diff
@@ -0,0 +1,50 @@
+%%
+% Copyright (c) 2017 - 2021, Pascal Wagler;
```
Comment (chrisfenner) [inline template/tcg-storage.tex:2]: Not needed since all contents of this file are new

File: template/tcg-storage.tex [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+% Copyright (c) 2014 - 2021, John MacFarlane
+%
+% All rights reserved.
```
Comment (chrisfenner) [inline template/tcg-storage.tex:5]: Should be covered by the repo LICENSE file so we don't have to add it for this file.

**Review by chrisfenner — APPROVED** [review]
I have only some very minor nits. Thanks for the change!

File: template/tcg-storage.tex (hunk 3 above)
Comment (joyshukla) [reply template/tcg-storage.tex]: this is removed now.

File: template/tcg-storage.tex (hunk 4 above)
Comment (joyshukla) [reply template/tcg-storage.tex]: I removed Copyright

File: Dockerfile (hunk 1 above)
Comment (joyshukla) [reply Dockerfile:141]: for now, we need this to be in Courier as all of TCG storage spec is Courier. We can remove this in future if we decide to not use courier.

Comment (chrisfenner) [reply Dockerfile:141]: Yup, changeable down the road easily.

**Review by chrisfenner — APPROVED** [review]

**Pull request merged**